account-keys/test-vectors/build.rs

```diff
@@ -0,0 +1,10 @@
+use mc_account_keys_test_vectors_structs::*;
+use mc_util_test_vectors::TestVectorWriter;
+
+fn main() {
+    println!("cargo:rerun-if-changed=build.rs");
```

Comment:
Is this not the default behavior already?

Reply:
The default behavior is to rebuild if any of the files in the crate change. This is an issue since the generated jsonl files are in a subdirectory of the crate and would cause it to rebuild every time. According to the cargo book, this is how you disable it:

> If the build script inherently does not need to re-run under any circumstance, then emitting cargo:rerun-if-changed=build.rs is a simple way to prevent it from being re-run.

https://doc.rust-lang.org/cargo/reference/build-scripts.html#cargorerun-if-changedpath

Reply:
> By default, it takes a conservative approach of always re-running the build script if any file within the package is changed (or the list of files controlled by the exclude and include fields).

https://doc.rust-lang.org/cargo/reference/build-scripts.html#change-detection

Reply:
This is worth mentioning in a comment for the next person who thinks this is redundant.